src/arch/gni/machine-onesided.C

```diff
@@ -367,6 +367,9 @@ void _performOneRputForWorkerThread(MSG_LIST *ptr) {
 
 void LrtsInvokeRemoteDeregAckHandler(int pe, NcpyOperationInfo *ncpyOpInfo) {
 
+  if(ncpyOpInfo->opMode == CMK_BCAST_EM_API)
```

Comment:
This should presumably be in a separate patch?

Reply:
This change of opMode being `CMK_BCAST_EM_API` was introduced because of the UCX layer requiring de-registration support. For that reason Gni, OFI and Verbs also had to support that opMode, where they had to simply not do anything (and hence return).

Reply:
Can you explain this in the body of the commit msg?

Reply:
Yes, I can.